Sources/Commandant/Command.swift

```diff
@@ -20,6 +20,9 @@ public protocol CommandProtocol {
 	/// `help`).
 	var verb: String { get }
 
+	/// Optional list of additional verbs which can be used to invoke this command.
+	var aliases: [String] { get }
```

Comment:
```suggestion
	var aliases: [String]? { get }
```

but let aliases be optional and not defined (null)